rewrite-weblogic/docs/recipes/jakarta-server-faces-3x.md

```diff
@@ -90,6 +93,9 @@ dependencies {
 ```
 2. Run `gradle rewriteRun` to run the recipe.
 
+> [!TIP]  
+> If you just want to dry run the recipe without changing the code, use `gradle rewrite:DryRun`.

```

Comment:
Same here: The correct command is `gradle rewriteDryRun`. Reference: https://docs.openrewrite.org/reference/gradle-plugin-configuration#the-dryrun-task